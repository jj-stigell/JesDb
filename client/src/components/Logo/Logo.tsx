import React from 'react'

export default function Logo (): React.JSX.Element {
  return (
    <h1
      className={`
        text-4xl text-center font-extrabold bg-gradient-to-r
        from-orange-500 to-yellow-300 text-transparent bg-clip-text`
      }
      role="img"
      aria-label="JesDb logo"
    >
      JesDb
    </h1>
  )
}
