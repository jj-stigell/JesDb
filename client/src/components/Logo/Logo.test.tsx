import React from 'react'
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Logo from './Logo'

describe('Logo Component', () => {
  it('should render the component with all props', () => {
    const { asFragment } = render(<Logo />)

    expect(screen.getByText('JesDb')).toBeInTheDocument()
    expect(asFragment()).toMatchSnapshot()
  })
})
