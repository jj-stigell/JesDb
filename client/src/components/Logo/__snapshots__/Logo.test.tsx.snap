// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should render the component with all props 1`] = `
<DocumentFragment>
  <h1
    aria-label="JesDb logo"
    class="
        text-4xl text-center font-extrabold bg-gradient-to-r
        from-orange-500 to-yellow-300 text-transparent bg-clip-text"
    role="img"
  >
    JesDb
  </h1>
</DocumentFragment>
`;
